.github/workflows/refapp-2x-legacy.yml

```diff
@@ -0,0 +1,38 @@
+name: RefApp 2.x Legacy Selenium Firefox
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  repository_dispatch:
+    types: [qa]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      firefox:
+        image: selenium/node-firefox:3.141.59-gold
+        env:
+          HUB_HOST: hub
+          HUB_PORT: 4444
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Checkout qaframework
+        uses: actions/checkout@master
+        with:
+          repository: ${{github.repository}}
+      - name: run db and web containers
+        run: |
+          docker-compose -f docker/docker-compose-refqa.yml up -d
+      - name: wait for openmrs instance to start
+        run: while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8080/openmrs/login.htm)" != "200" ]]; do sleep 1; done
+      - name: Run qaframework on selenium
+        run: |
+          mvn clean install -DskipTests=true
+          cp -f src/test/resources/org/openmrs/uitestframework/test-local.properties src/test/resources/org/openmrs/uitestframework/test.properties
+          cd src/main/java
```

Comment:
Thanks @kdaud , i meant, is this workflow triggering selenium tests specifically.

Reply:
`cd src/main/java` navigates the folder where the selenium legacy tests are residing and the command that is triggered does the building of the tests!

Reply:
Are we supposed to see a number of built selenium  tests with in the logs or the workflow is supposed to only run BDD written  tests ?

Reply:
According to this logs [here](https://github.com/openmrs/openmrs-contrib-qaframework/runs/4375756745?check_suite_focus=true)  , its showing still runTest class gets triggered with default workflows like  **refappSelenium** and i think this is what i was also trying to overcome, Am glad you are into this too and great work man.

Reply:
@sherrif10 what is your thought?

Reply:
As long as we allow the selenium tests gets triggered independently , i think we will be ready to go. we need to override runTest class not to be triggered since it comes along with npm scripts which we might not want with selenium tests.

Reply:
Most likely we need to trigger the selenium legacy tests via `mvn clean <params>`!